app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="col header2">
+  <div class="row">
+    <div class="col-4 col1">
+      <%= link_to root_path do %>
+        <%= image_tag 'back-arrow.png', alt: 'Back', class: 'back-icon' %>
+      <% end %>
+    </div>
+    <div class="col col22">LOG IN</div>
+    <div class="col col2">
+      <a href="#" id="login-button">Next</a>
```

Comment:
- **[OPTIONAL]** Next link or button was okay for the Register page, but according to the original design of the log in page, it should be the `Log In` button or link. Please 🙏🏽 correct the name od the button to be similar to the original design.

| Original design | Your design |
|---|---|
| ![image](https://github.com/danutibe07/BudgetApp/assets/58642949/d4d6958f-ed35-477f-8bcf-b4f8bbda8477) | ![image](https://github.com/danutibe07/BudgetApp/assets/58642949/d1f2ca7e-3f8b-4519-9fdb-a040bd09392e) |